pkg/csvparser/example_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2015 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
This references the wrong LICENSE file.

Maybe include the Go license in this folder?

Reply:
Or just mention https://golang.org/LICENSE

Reply:
Done, I added https://golang.org/LICENCE